.github/workflows/main.yml

```diff
@@ -0,0 +1,57 @@
+name: ci
+
+on:
+  push:
+
+jobs:
+  pytest:
+    runs-on: ubuntu-latest
+    container:
+      image: nat1405/niri:0.1
+      credentials:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Run pytest, in dragons env, in container.
+        run: |
+          conda run -n dragons pytest .
+
+  deploy:
+    runs-on: ubuntu-latest
+    needs: pytest
+    if: github.ref == 'refs/heads/dev' || github.ref == 'refs/heads/master'
```

Comment:
By the looks of it, this forces a contributor to issue the PR from the `master` or `dev` branches which is not always the case. My style is to issue PR from feature branches in my fork.

Reply:
Easy, removed in 9e2671ae.